src/components/AssetScanner/stories/AssetScanner.stories.tsx

```diff
@@ -11,6 +11,43 @@ import full from './full.md';
 import customNotificationsDoc from './notifications.md';
 import ocrRequestDoc from './ocr-request.md';
 
+const FileInputComponent = (props: any) => {
+  const [image, setImage] = useState('');
+
+  const onChange = async (e: SyntheticEvent) => {
```

Comment:
this should fix TS error 2 lines below
```suggestion
  const onChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
```

Reply:
Nope, it doesn't help to avoid TS errors